src/javascripts/components/events.js

```diff
@@ -1,7 +1,37 @@
-const initEvents = () => {
+import firebase from 'firebase/app';
```

Comment:
this file should be in a folder called events.  There should be an events.js file and an events.scss file in the folder.  Change the import statement paths as needed

Reply:
fixed

Reply:
done